code/modules/paperwork/fakepaperwork.dm

```diff
@@ -1286,15 +1286,23 @@ GLOBAL_LIST_EMPTY(Beucratic_triumps)
 	var/mob/living/okay = signed
 	okay.emote("faint")
 	okay.Sleeping(12 SECONDS)
-	to_chat(okay, span_danger("Something something ooooh"))
-	sleep(2)
-	to_chat(okay, span_danger("Yarba diva da! hah!"))
-	sleep(3)
-	to_chat(okay, span_danger("Fortnite..."))
-	sleep(3)
-	to_chat(okay, span_danger("mhm yea..."))
+	var/time
+	var/static/list/yarba = list(
+		span_danger("Something something ooooh"),
+		span_danger("Yarba diva da! hah!"),
+		span_danger("Fortnite..."),
+		span_danger("mhm yea..."),
+		)
+	for(var/word in yarba)
+		time = time + 3
+		addtimer(CALLBACK(src, PROC_REF(restore_order_say), okay, word), time)
 	okay.apply_status_effect(/datum/status_effect/debuff/paperwork_dread)
 
+/obj/item/paper/political_PM/bloodseal/proc/restore_order_say(mob/living/H, worb)
+	if(!worb)
+		return
+	to_chat(H, worb)
```

Comment:
Instead make a regular list, remove the first element and stop once its empty like

```
X/proc/say_list(mob/living/user, list)
	if(!length(list))
		return
	var/string = list[0]
	list -= string
	to_chat(user, string)
	say_list(user, list)
```